modules/channel-telegram/src/backend/index.ts

```diff
@@ -0,0 +1,53 @@
+import 'bluebird-global'
+import * as sdk from 'botpress/sdk'
+import Telegraf from 'telegraf'
+
+import { Config } from '../config'
+
+import { setupBot, setupMiddleware } from './client'
+import { Clients } from './typings'
+
+const clients: Clients = {}
+
+const onServerReady = async (bp: typeof sdk) => {}
+const onServerStarted = async (bp: typeof sdk) => {
+  setupMiddleware(bp, clients)
+}
+
+const onBotMount = async (bp: typeof sdk, botId: string) => {
+  const config = (await bp.config.getModuleConfigForBot('channel-telegram', botId)) as Config
+
+  if (config.enabled) {
+    const bot = new Telegraf(config.botToken)
+    clients[botId] = bot
+    bot.startPolling()
+    await setupBot(bp, botId, clients)
+  }
+}
+
+const onBotUnmount = async (bp: typeof sdk, botId: string) => {
+  const client = clients[botId]
+  if (!client) {
+    return
+  }
+
+  client.stop()
+  delete clients[botId]
+}
+
+const entryPoint: sdk.ModuleEntryPoint = {
+  onServerStarted,
+  onServerReady,
+  onBotMount,
+  onBotUnmount,
+  definition: {
+    name: 'channel-telegram',
+    menuIcon: 'chrome_reader_mode',
```

Comment:
I imagine that chrome_reader_mode is not the telegram icon... if yes then that's a pretty bad naming, otherwise can you change this for the telegram icon. [FaTelegramPlane](https://react-icons.netlify.com/#/icons/fa)